package.json

```diff
@@ -72,8 +72,8 @@
     "zone.js": "0.8.17"
   },
   "peerDependencies": {
-    "@angular/common": "^2.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0",
-    "@angular/core": "^2.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0",
-    "nouislider": "^9.0.0 || ^10.0.0 || ^11.0.0"
+    "@angular/common": "^2.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0",
```

Comment:
Alternatively, I don't see any danger in merely specifying `>=2.x` for angular and `>=9.x` for nouislider

Reply:
Yes, I will change it to >=2.x